scripts/v.dissolve/v.dissolve.py

```diff
@@ -33,36 +33,339 @@
 # %end
 # %option G_OPT_V_OUTPUT
 # %end
+# %option G_OPT_DB_COLUMN
+# % key: aggregate_column
+# % description: Name of attribute columns to get aggregate statistics for
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_method
```

Comment:
Is singular better than plural? I'm not sure. Always my dilemma :-(

Reply:
Not sure either about plural versus singular in option names.

Reply:
I went with plural as in  `v.db.select ... columns=...`.